test_utils/src/sequencer.rs

```diff
@@ -76,41 +79,38 @@ impl Sequencer {
 }
 
 /// The core of the sequencer, totally ordering input bytes.
-pub struct SequencerCore<Message> {
+pub struct SequencerCore {
     /// Receive users' certificates to sequence
-    rx_input: Receiver<Message>,
-    /// Receive subscribers to update with the output of the sequence.
-    rx_subscriber: Receiver<Sender<(Message, SequenceNumber)>>,
+    rx_input: Receiver<Bytes>,
+    /// Communicate with subscribers to update with the output of the sequence.
+    rx_subscriber: Receiver<SubscriberMessage>,
     /// The global consensus index.
     consensus_index: SequenceNumber,
+    /// Holds all consensus outputs.
+    all_outputs: Vec<ConsensusOutput>,
```

Comment:
This is going to get pretty full pretty soon, no? How about we use our typed_store here down the line?

Reply:
It really depends what we want to do with the sequencer. In my mind it is just a testing facility used for unit tests (not even integration tests). I never considered it to be running as part of the dev/test net. If this is the idea, then it needs to change drastically.